### PR TITLE
Add Tests and Methods to Satisfy the Vehicle Requirements

### PR DESCRIPTION
**Pull request by memcmahon**
Implemented the following methods on Vehicle:
* speeding?
* speed
* num_adults

Questions for reviewer: 
* For the num_adults method, do you think there is an opp for refactor?

### CODE REVIEW

**Comment by nicktu12** [comment]
Looks good! Im gonna take a gander at the num adults method

File: lib/vehicle.rb [hunk 1]
```diff
@@ -11,4 +11,22 @@ def initialize(year, make, model)
...
+    def speed
+        @speeding = true
+    end
+
+    def add_passenger(passenger)
+        @passengers << passenger
+    end
+
+    def num_adults
+        @passengers.count do |passenger|
```
Comment (nicktu12) [inline lib/vehicle.rb:28]: I think count works great! Is there a find_all or filter type method that could work too? LGTM either way.

Comment (nicktu12) [reply lib/vehicle.rb:28]: Added some code in my PR that might help here too!

Comment (memcmahon) [reply lib/vehicle.rb:28]: notes from discussion:
* we could use passenger#adult? method to determine age

**Comment by nicktu12** [comment]
Nice new changes! Lets merge this

**Pull request merged**